Android/app/src/main/java/io/github/project_travel_mate/destinations/description/CityMapActivity.java

```diff
@@ -71,4 +74,12 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             manager.downloadAreaAsync(this, points, 14, 29);
         });
     }
+
+    @Override
```

Comment:
these are already merged.

Reply:
So you can approve my pull request?

Reply:
?

Reply:
@Florence-Njeri I have already added comments (and what needs to be changed) on your PRs.
Please do that and then, I'll approve PRs :)